conda-env/ci.yml

```diff
@@ -6,22 +6,19 @@ channels:
 dependencies:
   # Base
   # ==================
-  - python>=3.8
+  - python>=3.9
   - pip
-  - nco
   - cmor>=3.6.0
-  - tqdm
+  - dask
+  - nco
+  - netcdf4
+  - numpy
   - pyyaml
+  - tqdm
   - xarray
-  - netcdf4
-  - dask
-  - scipy
```

Comment:
`scipy` is a collection of multiple libraries including `numpy`. We only need `numpy` so `scipy` is replaced with `numpy`.